index.html

```diff
@@ -3404,9 +3404,8 @@ <h2>Definition of Roles</h2>
 			<rdef>generic</rdef>
 			<div class="role-description">
 				<p>A nameless container <a>element</a> that has no semantic meaning on its own.</p>
-				<p>Contrast with <rref>group</rref>, which semantically groups its descendants in a named container.</p>
-				<p>A <code>generic</code> can provide a limited number of accessible states and properties for its descendants, such as <pref>aria-live</pref> attributes. This differentiates it from the <rref>presentation</rref> role.</p>
-				<p>The <code>generic</code> role is intended for implementors of User Agents. Authors SHOULD NOT use this role in content.</p>
+				<p>The <code>generic</code> role is intended for implementors of User Agents. Authors SHOULD NOT use this role in content. Authors MAY use <rref>presentation</rref> or <rref>none</rref> to remove implicit accessibility semantics, or a semantic container role, such as <rref>group</rref>, to semantically group descendants in a named container.</p>
```

Comment:
The change LGTM as-is, but perhaps also change this clause: "intended for implementors of User Agents."

I suggest: "intended for use as the default computed role of generic elements in host languages (e.g. `<span> or `<div>`)." 

This is partially for UA implementors, but for compatibility testing purposes, including web author tests using WebDriver... Not only implementors. If you prefer, we could file a separate issue to address this minor prose change.

Reply:
Nit: don't capitalize "User Agents"

Nit: The "Authors MAY" sentence seems rather commaful. Are the commas needed in between "role" and "such" and after the "</rref>"?

Reply:
@cookiecrook I added your new "intended for use as..." words (changed "default computed role" to "implicit role" because that seems to be the standard lingo). I wanted to keep the "implementors of user agents" to reinforce to authors that they really have no business setting role=generic on anything...

I will paste the full 3 paragraphs below - please have a look if you have a moment.

@joanmarie "user agents" it is, and I removed 2 commas that I agree were excessive.